docs/hooks-nodejs.md

```diff
@@ -188,6 +188,25 @@ before("Machines > Machines collection > Get Machines", function (transaction) {
   transaction.request.body = JSON.stringify(requestBody);
 });
 ```
+### Modifying Multipart Transaction Request Body Prior to Execution
+
+```javascript
+const hooks = require('hooks');
+const Multipart = require('multi-part');
```

Comment:
Is this standard Node.js library or is it something user needs to install? dtto with `stream-to-string`. The examples should use standard library as much as possible, or at least mention any dependencies explicitly.